libs/components/router/testing/src/href/href-harness.ts

```diff
@@ -0,0 +1,42 @@
+import { HarnessPredicate } from '@angular/cdk/testing';
+import { SkyComponentHarness } from '@skyux/core/testing';
+
+import { SkyHrefHarnessFilters } from './href-harness-filters';
+
+export class SkyHrefHarness extends SkyComponentHarness {
+  static hostSelector = '[skyHref]';
+
+  /**
+   * Gets a `HarnessPredicate` that can be used to search for a
+   * `SkyHrefHarness` that meets certain criteria.
+   */
+  public static with(
+    filters: SkyHrefHarnessFilters
+  ): HarnessPredicate<SkyHrefHarness> {
+    return this.getDataSkyIdPredicate(filters);
+  }
+
+  async getHref(): Promise<string | null> {
+    return this.forceStabilize().then(async () => {
+      return await (await this.host()).getAttribute('href');
+    });
+  }
+
+  async getLinkText(): Promise<string> {
+    return this.forceStabilize().then(async () => {
+      return await (await this.host()).text();
+    });
+  }
+
+  async isVisible(): Promise<boolean> {
+    return this.forceStabilize().then(async () => {
+      return await (await this.host())
+        .matchesSelector('[hidden]')
+        .then((hidden) => !hidden);
+    });
+  }
+
+  async isLinked(): Promise<boolean> {
```

Comment:
I'd consider a different name, such as `hasHref` to be a bit more specific. "Linked" could mean various things (e.g. "linked to what?"). 

Also, since we already have the `getHref` method, I'm curious about what the use case is for this method?

Reply:
Dropped this method.